### ISSUE TITLE
Unable to change table name (0.29.0.SNAPSHOT)

### ISSUE THREAD

**Issue by MarkRx** [metabase/metabase#7175]
Using the latest 0.29.0 snapshot 87565f4 the following error occurs attempting to change the name of a table created in a previous version (changing the name of an existing table seems to work fine). The data in the metabase metadata database has an entity_type of entity/GenericTable.

> 03-21 13:38:19 DEBUG metabase.middleware :: PUT /api/table/372 400 (639 ?s) (0 DB calls).
> --
>   | {:errors {:entity_type "value may be nil, or if non-nil, value must be one of: `event`, `person`, `photo`, `place`."}}

POST /api/table/372
```
{
	"description": null,
	"entity_type": "entity/GenericTable",
	"schema": "MBASE",
	"raw_table_id": null,
	"show_in_getting_started": false,
	"name": "table",
	"caveats": null,
	"rows": 407407,
	"updated_at": "2018-03-20T19:50:04.000Z",
	"entity_name": null,
	"active": true,
	"db_id": 4,
	"visibility_type": null,
	"display_name": "table",
	"created_at": "2018-02-08T19:41:48.000Z",
	"points_of_interest": null,
	"metadataStrength": 0.43636363636363634
}
```

https://github.com/metabase/metabase/blob/master/docs/api-documentation.md

**Comment by senior** [comment]
This was fixed by https://github.com/metabase/metabase/pull/7277 if you'd like to update your build to the latest, it should work.

The fix will be included in the next release (0.29.0)

**Comment by MarkRx** [comment]
I can confirm it has been fixed. Thanks.
